docs/client-specifications.rst

```diff
@@ -0,0 +1,266 @@
+* the Remote Settings team has to be consulted to validate the implementation
+
+
+Existing Clients
+----------------
+
+As of April 2024:
+
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+|                        | Gecko | Rust Client | application-services/remote-settings | kinto-http.py | kinto.js |
++========================+=======+=============+======================================+===============+==========+
+| Write Operations       | ❌    | ✅          | ❌                                   | ✅            | ✅       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+| Add/Remove Attachments | ❌    | ✅          | ❌                                   | ❌            | ✅       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+| Approve/Reject reviews | ❌    | ✅          | ❌                                   | ❌            | ❌       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+| Changesets Endpoints   | ✅    | ✅          | ❌                                   | ✅            | ❌       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+| Cache Busting          | ✅    | ✅          | ❌                                   | ✅            | ❌       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+| Env Switching          | ✅    | ~ Partial   | ✅                                   | ❌            | ❌       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+| Signature Verification | ✅    | ✅ Optional | ❌                                   | ❌            | ❌       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+| Local State            | ✅    | ✅ Optional | ❌                                   | ❌            | ✅       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+| Fetch Attachments      | ✅    | ✅          | ✅ Without integrity check           | ❌            | ✅       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+| Backoff                | ✅    | ✅          | ✅                                   | ✅            | ✅       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
+| Deprecation            | ❌    | ❌          | ❌                                   | ✅ Minimal    | ✅       |
++------------------------+-------+-------------+--------------------------------------+---------------+----------+
```

Comment:
This is an improvement from the last time I looked, but still looking off:
<img width="893" alt="image" src="https://github.com/mozilla/remote-settings/assets/20747904/57fdf485-971a-4000-8fd3-4a50112f05e1">

Don't know how much you want to keep fiddling with this, and it might be Github's rendering?

Reply:
Seems fine with Sphinx 

<img width="695" alt="Screenshot 2024-04-16 at 20 02 39" src="https://github.com/mozilla/remote-settings/assets/546692/37a6ae4d-a8be-4a0d-a50a-4eea5233a9c7">

Reply:
<img width="963" alt="Screenshot 2024-04-16 at 20 05 11" src="https://github.com/mozilla/remote-settings/assets/546692/8392b266-e8e6-4962-a9f0-b580060c3d5e">

so does my terminal

Reply:
Excellent, it's Github's fault, then 😏